crates/adapters/src/server/mod.rs

```diff
@@ -242,6 +244,19 @@ pub struct ServerArgs {
     /// automatically
     #[arg(short = 'p', long)]
     default_port: Option<u16>,
+
+    /// Whether to enable TLS on the HTTP server.
+    /// Iff true, is it allowed and required to set `https_tls_cert_path` and `https_tls_key_path`.
+    #[arg(long, default_value_t = false)]
+    pub enable_https: bool,
```

Comment:
I wonder if this can be dropped and instead have 

    #[arg(long, requires = "https_tls_key_path")]
    #[arg(long, requires = "https_tls_cert_path")]

(if they are None doesn't it imply enable_https =false?)

Reply:
Hmm, I thought a flag would make it clearer, but it is indeed redundant